src/server/pbsd_main.c

```diff
@@ -1291,6 +1291,10 @@ main(int argc, char **argv)
 	CLEAR_HEAD(svr_exechost_startup_hooks);
 	CLEAR_HEAD(svr_execjob_attach_hooks);
 	CLEAR_HEAD(svr_allscheds);
+#ifdef WIN32
+	if(!passwd_cache_init)
+		CLEAR_HEAD(passwd_cache_ll);
```

Comment:
Why are you even checking passwd_cache_init? Why not simply call CLEAR_HEAD(passwd_cache_ll)? Just like how all the other lists are being initialized?

Reply:
Okay